src/components/Stakeholders/StakeholderDetail/List.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useContext } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Paper, IconButton, Grid, ListItem } from '@material-ui/core';
+import UnlinkIcon from '@material-ui/icons/LinkOff';
+import LinkIcon from '@material-ui/icons/Link';
+import { StakeholdersContext } from '../../../context/StakeholdersContext';
+import ParentChildListItem from './ParentChildListItem';
+
+const useStyles = makeStyles({
+  listItem: {
+    backgroundColor: 'white',
+    marginBottom: 4,
+    borderRadius: 4,
+  },
+});
+
+function StakeholderList({ id, data, type, linked }) {
+  const classes = useStyles();
+  const [isLinked, setIsLinked] = useState(linked);
+  const { deleteStakeholder } = useContext(StakeholdersContext);
+
+  const handleChange = (e, data) => {
+    setIsLinked(!isLinked);
+    const confirmed = window.confirm(
+      'Are you sure? Continuing will delete the stakeholder from the system.'
```

Comment:
If we're making this a delete function rather than unlink, let's use a trash icon.
If deletion is just a placeholder for unlinking, can you add a comment to explain what's happening here?